files/acl.auth.php

```diff
@@ -0,0 +1,61 @@
+# acl.auth.php
+# <?php exit()?>
+# Don't modify the lines above
+#
+# Access Control Lists
+#
+# Editing this file by hand shouldn't be necessary. Use the ACL
+# Manager interface instead.
```

Comment:
This is a good position to add {{ansible_managed}}

Reply:
done